### ISSUE TITLE
Show time difference / change from previous slice

### ISSUE THREAD

**Issue by kthornbloom** [Ultimaker/Cura#10481]
### Is your feature request related to a problem?

After (hopefully) optimizing a bunch of settings, I re-slice to see if it's faster. Sometimes I forgot to note the previous time, and I'm not sure if the new settings are better or worse.

### Describe the solution you'd like

Each slice, show a time difference with the previous slice:

![example](https://user-images.githubusercontent.com/3322795/133934558-a9c40aae-b24b-4604-b4d7-4f240f7e1a07.png)

### Describe alternatives you've considered

I've tried remembering the old time estimate, but my brain no work good.

### Affected users and/or printers

Errybody.

**Comment by Vandresc** [comment]
Hi @kthornbloom , thank you for your feedback, I will bring your request up to the team.

**Comment by Vandresc** [comment]
Devs for our own reference, see CURA-8583 in our backlog.

**Comment by fvrmr** [comment]
We are constantly working on making Ultimaker Cura better. Our community is a big part of that by requesting new features and reporting bugs. 

To be able to focus on the most requested and needed features from our community and from Ultimaker, we have decided to remove this feature request from our backlog. We believe we will not work on this anytime soon and will therefore defer this issue.

Since Cura is open source, you or anyone else is more than welcome to work on this feature and create a pull request yourself.

**Comment by kthornbloom** [comment]
Yes, this is absolutely a feature I would still like to see happen. I think it would be extremely easy to implement (just some addition/subtraction here), and would be very useful to see at a glance what you've been able to optimize. 

Here's a fresh UI demo:

<img width="720" alt="Screenshot 2023-06-29 at 10 12 47 AM" src="https://github.com/Ultimaker/Cura/assets/3322795/a3aeaa48-094e-4268-b26e-cf1eb301b78d">

**Comment by MariMakes** [comment]
Quick update from our side 👋

Thanks for upvoting this issue again. I showed it to the team and they are still excited to work on this, we have new insights that might make implementation of this easier. We've added a ticket to the backlog with the intent to improve this. 
For internal reference CURA-10897

Thanks for the report! 👍

**Comment by kthornbloom** [comment]
Thanks for the update! Looking forward to it
